app/config.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+import os
+
+import yaml
+
+
+def read_config():
+    config_path = Path(os.environ.get('CONFIG_PATH', './../config.yml')).resolve()
+    try:
+        with open(config_path) as f:
+            return yaml.safe_load(f)
+    except Exception as e:
+        msg = "Could not load config from path {}: {}".format(config_path, str(e))
+        raise Exception(msg)
```

Comment:
Consider creating a custom exception i.e. AutoscalerException

Catch local exceptions, log and re-throw as custom exceptions, the next level up then catches the custom exception